ewb_api_integration/hooks.py

```diff
@@ -143,3 +143,17 @@
 #
 # auto_cancel_exempted_doctypes = ["Auto Repeat"]
 
+doctype_js = {
+    "Sales Invoice" : "public/js/sales_invoice.js",
+}
+
+doc_events = {
+	"Sales Invoice": {
+		"on_submit": "ewb_api_integration.ewb_api_integration.doctype.ewb_api_integration_settings.ewb_api_integration_settings.set_field_values",
+		"on_update_after_submit": "ewb_api_integration.ewb_api_integration.gsp.gsp.update_transporter",
```

Comment:
error should prevent update.

Reply:
```on_update_after_submit``` itself prevents the update incase of any error.

Reply:
Can you review the below commits? It will resolve the above.